### ISSUE TITLE
Bug list v4.0.0

### ISSUE THREAD

**Issue by csabdulahad** [csabdulahad/hati#11]
### ~~This should be the check for cli detection:~~ ✅

```php
<?php
$sAPI = php_sapi_name();
if ($sAPI === 'cli' || $sAPI === 'phpdbg') {
    // it is in CLI
}
```
### ~~We don't need to include link tag for jst library as it is now done by the library itself~~ ✅

**Comment by csabdulahad** [comment]
Fixed in v5.0.0

**Comment by csabdulahad** [comment]
All bug fixed in v5.0.0 release
